src/root.jl

```diff
@@ -193,16 +193,20 @@ on type `T` and jagg type `J`.
 In order to retrieve data from custom branches, user should defined more speialized
 method of this function with specific `T` and `J`. See `TLorentzVector` example.
 """
+function interped_data(rawdata, rawoffsets, ::Type{T}, ::Type{Nojagg}) where {T<:Bool}
```

Comment:
Bool is a concrete type
```suggestion
function interped_data(rawdata, rawoffsets, ::Type{Bool}, ::Type{Nojagg})
```